charts/spire/README.md

```diff
@@ -7,6 +7,51 @@ A Helm chart for deploying the complete Spire stack including: spire-server, spi
 
 **Homepage:** <https://github.com/spiffe/helm-charts/tree/main/charts/spire>
 
+## Install notes
+
+To do a quick non production install suitable for quick testing in something like minikube:
+
+```shell
+helm install spire-crds --repo https://spiffe.github.io/helm-charts-hardened/
+helm install spire --repo https://spiffe.github.io/helm-charts-hardened/
```

Comment:
```suggestion
helm install spire --repo https://spiffe.github.io/helm-charts-hardened/ --create-namespace
```

Reply:
Its installing to default now, which will always exist.

Reply:
But if you still want it there just in case, it should probably be on the crds one, as it runs first?

Reply:
sorry i thought default namespace was `spire`, not `default`. 🤦 . You had it right the first time.

Reply:
updated the suggested change.

Reply:
kk. crds should be in the same place... I'll update it all and then ping you back.